optprompt/optprompt.py

```diff
@@ -37,7 +36,7 @@ def parse_args(self, *args, **kwargs):
         return opts
 
     def parse_configs(self, opts):
-        if not self.config_files:
+        if not any(os.path.isfile(f) for f in self.config_files):
```

Comment:
I like this but I think it should be slightly more helpful when it finds an issue. Ideally it should return if there are no config files provided but raise an exception if there is one in the list that doesn't exist.

Reply:
Raising an exception when the files were not present is what I am specifically preventing by doing this. The toml library was raising an exception if none of the files existed. Since this library is using the toml files as configuration preferences, there is no reason they need to exist. Same thing as deleting everything in my home directory; none of my apps should stop working in Linux. -- https://askubuntu.com/questions/55205/is-it-safe-to-delete-every-hidden-folder-in-the-home-directory